crates/kitsune_p2p/dht_arc/src/dht_arc/dht_arc_bucket.rs

```diff
@@ -93,8 +103,8 @@ impl DhtArcBucket {
         }
     }
 
-    /// Get the density of this bucket.
-    pub fn density(&self) -> PeerDensity {
+    #[deprecated = "use peer_view"]
```

Comment:
Why does this PR introduce a deprecated function?
Is this for cases where you don't want people to choose the version of peer view?